targetconfig.json

```diff
@@ -372,21 +372,33 @@
     "kiosk": {
         "games": [
             {
-                "id": "05854-11479-55466-38029",
-                "name": "MakeCode Racers",
-                "description": "Top-down racing game for 1-4 players. Press Left and Right to steer, A to use picked up item.",
+                "id": "25572-25731-72869-29244",
+                "name": "Space Destroyer",
+                "description": "Use the lasers on your spaceship to shoot falling asteroids!",
+                "highScoreMode": "SingleAscending"
+            },
+            {
+                "id": "84490-94874-87123-99597",
+                "name": "Bunny Hop!",
+                "description": "Help your bunny hop over obstacles as they run through the forest",
+                "highScoreMode": "SingleAscending"
+            },
+            {
+                "id": "60307-91678-81892-80686",
```

Comment:
@Jaqster, I swapped this game id for its non-persistent version. Let me know if you want it changed back.